templates/frontend/components/header.tpl

```diff
@@ -43,11 +43,13 @@
 					{else}
 						<div class="pkp_site_name">
 					{/if}
-						{if $currentContext && $multipleContexts}
-							{url|assign:"homeUrl" page="index" router=$smarty.const.ROUTE_PAGE}
-						{else}
-							{url|assign:"homeUrl" context="index" router=$smarty.const.ROUTE_PAGE}
-						{/if}
+						{capture assign="homeUrl"}
+							{if $currentContext && $multipleContexts}
+								{url page="index" router=$smarty.const.ROUTE_PAGE}
+							{else}
+								{url context="index" router=$smarty.const.ROUTE_PAGE}
+							{/if}
+						{/capture}
```

Comment:
Is there a scenario where line breaks in a `href` attribute can cause issues? Should `{strip}` be used here? Or can we print `$homeUrl|trim` below maybe?

Reply:
Smarty seems to remove whitespace when there's just a Smarty tag on the line. I think we're OK as is.